.github/workflows/build-production.yaml

```diff
@@ -7,6 +7,38 @@ on:
     types: [closed]
 
 jobs:
+  tag_release:
+    name: Tag and Release
+    # only run if PR is merged (not just closed)
+    if: github.event.pull_request.merged
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+        # the tag will match the package.json version (eg. v1.0.0)
+      - name: Tag
+        id: autotagger
+        uses: butlerlogic/action-autotag@stable
+        with:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          strategy: package
+          prefix: v
+          # use the body of the PR commit as the tag message
+          tag_message: ${{ github.event.pull_request.body }}
+
+      - name: Release
+        id: create_release
+        if: steps.autotagger.outputs.tagname != ''
+        uses: actions/create-release@v1.0.0
```

Comment:
Note: This `actions/create-release` is currently archived and not maintained, I think it's ok to use it, but in the future we might consider moving over to one of the maintained actions mentioned in the docs https://github.com/actions/create-release

Reply:
created a Jira ticket here: https://jira.nypl.org/browse/SFR-1568